share/completions/git.fish

```diff
@@ -433,6 +433,31 @@ complete -f -c git -n '__fish_git_using_command merge' -l abort -d 'Abort the cu
 
 # TODO options
 
+### mergetool
+
+function __fish_git_mergetools
+    set -l tools diffuse diffmerge ecmerge emerge kdiff3 meld opendiff tkdiff vimdiff gvimdiff xxdiff araxis p4merge bc codecompare
+    for tool in $tools
+        command --search $tool >/dev/null; and begin
```

Comment:
Since you are doing `begin`/`end` anyway, this would be nicer with `if command`.